setup.py

```diff
@@ -0,0 +1,39 @@
+import os
+from setuptools import setup, find_packages
+
+
+def src(pth):
+    return os.path.join(os.path.dirname(__file__), pth)
+
+
+# Project description
+descr = 'Python library implementing linear operators with MPI'
+
+# Setup
+setup(
+    name='pylops_mpi',
+    description=descr,
+    long_description=open(src('README.md')).read(),
+    long_description_content_type='text/markdown',
+    keywords=['algebra',
+              'inverse problems',
+              'large-scale optimization'],
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```

Comment:
Change this into `Development Status :: 1 - Planning` for now

Reply:
Done...